File: ManejoArchivosJava/src/ManejoArchivosJava/Vehiculo.java
/*
 *esta es la super clase del proyecto en esta, se crean algunos atributos 
como marca, modelo y color; ademas de  que se crea el respectivo constuctor para esta clase  
 */
package ManejoArchivosJava;

/**
 * 28 septiembre 2018
 * @author Manuela García Monsalve 
 */
public abstract class Vehiculo {
 
    String marca;
    String modelo;
    String color;
    // constructor de la clase vehiculo 
    public Vehiculo(String marca,String modelo,String color)
    {
        this.marca = marca;
        this.modelo = modelo;
        this.color = color;
    }
    abstract String VerDatos();
}

File: ManejoArchivos/Vehiculo.py
#Autor Manuela Garcia Monsalve
# 28 septiembre 2018
#Esta es la super clase Vehiculo donde se encuentran los atributos de marca, modelo y color que seran
#heredados para las subclases

class Vehiculo():

    def __init__(self,marca,color, modelo): #Se generan los atributos para poder ser heredados
        self.marca = marca
        self.color = color
        self.modelo = modelo


    def Prender(self): #Metodo prender vehiculo
        pass

    def Arrancar(self):#Metodo arrancar vehiculo
        pass

    def Apagar(self):#Metodo apagar vehiculo
        pass


